xlators/mgmt/glusterd/src/glusterd-statedump.c

```diff
@@ -216,19 +214,6 @@ glusterd_dump_priv(xlator_t *this)
         /* Dump peer details */
         GLUSTERD_DUMP_PEERS(&priv->peers, uuid_list, _gf_false);
 
-        /* Dump pmap data structure from base port to last alloc */
```

Comment:
nit: we may want to dump the port, where is it done?

Reply:
In that case, we will have to dump the ports using the brickinfo struct, by traversing through the brick_list.